crates/sparrow-backend/src/snapshots/sparrow_backend__logical_to_physical__tests__logical_to_physical_arithmetic-3.snap

```diff
@@ -59,7 +59,7 @@ steps:
       - name: input
         literal_args: []
         args: []
-        result_type: Float64
+        result_type: "Null"
```

Comment:
I think it's because the merge currently doesn't set the merged type correctly in `logical_to_physical`. See:

```
let merged_step = self.plan.get_or_create_step_id(
    sparrow_physical::StepKind::Merge,
    inputs.clone(),
    ExprVec::empty(),
    DataType::Null,
);
```